Angular: add info about standalone components (#7321)
DevExpress/devextreme-documentation: includes/ng-import-devextreme-modules.md

DevExtreme components are [standalone](https://angular.dev/guide/components). Import them as follows:

    <!-- tab: app.component.ts -->
    import { Component } from '@angular/core';
    import { DxButtonComponent } from 'devextreme-angular/ui/button';

    @Component({
        selector: 'app-root',
        standalone: true,
        imports: [DxButtonComponent],
        templateUrl: './app.component.html',
        styleUrl: './app.component.css'
    })
    export class AppComponent { }
    

> The nested components should be imported as well. For example, the following code snippet imports DataGrid, its columns, sorting, and a template.

Reviewer: ```suggestion
The nested components should also be imported. For example, the following code snippet imports DataGrid, its columns, sorting, and a template.
```

[note] If you use `*dxTemplate`, import `DxTemplateDirective`.

    <!-- tab: app.component.ts -->
    import { Component } from '@angular/core';
    import { DxTemplateDirective } from 'devextreme-angular';
    import { DxDataGridComponent, DxiDataGridColumnComponent } from 'devextreme-angular/ui/data-grid';

    @Component({
        selector: 'app-root',
        standalone: true,
        imports: [DxDataGridComponent, DxiDataGridColumnComponent, DxTemplateDirective],
        templateUrl: './app.component.html',
        styleUrl: './app.component.css'
    })
    export class AppComponent { }

    <!-- tab: app.component.html -->
    <dx-data-grid>
        <dxo-data-grid-sorting mode="none"></dxo-data-grid-sorting>
        <dxi-data-grid-column dataField="date"></dxi-data-grid-column>
        <dxi-data-grid-column 
            caption="Employee"
            cellTemplate="employeeTemplate"
        ></dxi-data-grid-column>
        <div *dxTemplate="let data of 'employeeTemplate'">
            <img
                [src]="data.value"
                alt="Picture of {{ data.data.FirstName }} {{ data.data.LastName }}"
            />
        </div>
    </dx-data-grid>

> Modules help when dealing with multiple configuration components. You can import them as follows:

Reviewer: ```suggestion
Modules facilitate working with multiple configuration components. You can import them as follows:
```
